### ISSUE TITLE
E: You don't have enough free space in /var/cache/apt/archives/

### ISSUE THREAD

**Issue by philgree** [whiskerz007/proxmox_hassos_install#14]

**Comment by philgree** [comment]
Thanks for the commands to switch to the community repos and remove the nag. I've executed both, checked for upgrades and found 32 updates. I'm getting a message that I don't have sufficient free space.

In theory, I should have lots of available disk space on the machine. Is there a way to increase the space allocated to Proxmox itself?

**Comment by philgree** [comment]
Looks like I only allocated 100Gb of 500 to Proxmox and the 100Gb is 99.7% full so what I'm really needing to do is understand how to assign the remainder of the disk so Proxmox can use it

**Comment by whiskerz007** [comment]
@philgree By default, Proxmox will will setup two primary storage areas. One should be used for Proxmox and data needed to setup a VM or LXC. The other should be used for the virtual disks for the VMs and LXCs. Check the storage location for the VMs and LXCs, if they are using `local` consider moving them to the other storage pool. To help with the setup of new VMs and LXCs, I would change the `Content` parameter in the `local` storage pool to equal `VZDump backup file, ISO image, Container template`.

**Comment by philgree** [comment]
@whiskerz007 Thank you very much. I struggled with this problem for a few days. Ultimately my Googlefu failed me, I went back to a bare metal install and it setup exactly as you describe. I obviously did something strange on my first install - I remember specifying just a 100Gb partition to 'dip my toe in the water' rather than letting it take the whole disk.

As is so often the case, messing up is a great way to learn! I'm sure I (and others) will pass this way again.
